app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
`.gitignore` is a powerful file that let us control what kind of files are added to source control.
I noticed that you only add the build folder to be excluded in git repository.

Maybe you can take a look to the below suggested `.gitignore` file and add all necessary rules to it.

```suggestion
# Built application files
/*/build/

# Crashlytics configuations
com_crashlytics_export_strings.xml

# Local configuration file (sdk path, etc)
local.properties

# Gradle generated files
.gradle/

# Signing files
.signing/

# User-specific configurations
.idea/libraries/
.idea/workspace.xml
.idea/tasks.xml
.idea/.name
.idea/compiler.xml
.idea/copyright/profiles_settings.xml
.idea/encodings.xml
.idea/misc.xml
.idea/modules.xml
.idea/scopes/scope_settings.xml
.idea/vcs.xml
*.iml

# OS-specific files
.DS_Store
.DS_Store?
._*
.Spotlight-V100
.Trashes
ehthumbs.db
Thumbs.db
```